templates/comment_form/author_field.twig

```diff
@@ -1,5 +1,14 @@
-<div class="form-group animated-label">
-	{% set your_name = __( 'Your Name', 'planet4-master-theme' ) %}
-	<input type="text" class="form-control" id="comments-name-input" name="author" placeholder="{{ your_name }}" required />
+<div class="form-group animated-label comment-author">
+	{% set placeholder = __( 'Your Name', 'planet4-master-theme' ) %}
+	{% set commenter_name = function('wp_get_current_commenter').comment_author ?? '' %}
```

Comment:
This is the fix for the case where the "Cookies opt-in" checkbox is enabled in the backend and checked by the commenter